scripts/ci/run-integration-tests

```diff
@@ -44,10 +44,11 @@ function runTestFile {
   exit 1
 }
 
+find $PROJECT_ROOT/tests -type f -name '*-cy.js' | while read file; do runTestFile "$file"; done
 
-for f in $PROJECT_ROOT/tests/**/*-cy.js; do
-  runTestFile "$f"
-done
+# for f in $PROJECT_ROOT/tests/**/*-cy.js; do
+#   runTestFile "$f"
+# done
```

Comment:
shouldn't it be removed instead of commented out? Same for other instances on this file.

Reply:
oh, yes. good catch! will fix it tomorrow when resolving the conflicts.